input/listen.go

```diff
@@ -53,21 +55,23 @@ func acceptTcp(addr string, listener *net.TCPListener, handler Handler) {
 		log.Notice("error accepting on %v/tcp, closing connection", addr)
 		l.Close()
 
-		backoff := time.Duration(time.Second)
+		b := &backoff.Backoff{
+			Min: 500 * time.Millisecond,
+		}
 		for {
 			log.Notice("reopening %v/tcp", addr)
 
 			l, err = listenTcp(addr)
 			if err == nil {
+				b.Reset()
```

Comment:
If we're going to Reset() then we should move the backoff initialization outside the loop, we'll also want to add a reasonable max, I'm thinking 1 minute

Reply:
Updated